lib/livebook_web/live/output.ex

```diff
@@ -243,19 +244,31 @@ defmodule LivebookWeb.Output do
     )
   end
 
-  defp render_output({:error, formatted}, %{}) do
-    assigns = %{message: formatted}
+  defp render_output({:error, formatted, {:unavailable_env, "LB_" <> _rest}}, %{
+         socket: socket,
+         session_id: session_id
+       }) do
+    assigns = %{message: String.replace(formatted, ~s(environment variable "LB_), ~s(secret "))}
 
     ~H"""
-    <div
-      class="whitespace-pre-wrap font-editor text-gray-500"
-      role="complementary"
-      aria-label="error"
-      phx-no-format
-    ><%= ansi_string_to_html(@message) %></div>
+    <div class="flex justify-between">
```

Comment:
Is the button side by side with the error? It's probably a better use of space if it's below the error, wdyt?

Reply:
Yep! @josevalim it's up to you

Reply:
@cristineguadelupe the button should be visible as up as possible.